src/Resources/skeleton/crud/controller/Controller.tpl.php

```diff
@@ -0,0 +1,104 @@
+<?= "<?php\n"; ?>
+
+namespace <?= $namespace; ?>;
+
+use <?= $entity_full_class_name; ?>;
+use <?= $form_full_class_name; ?>;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @Route("<?= $route_path; ?>", name="<?= $route_name; ?>_")
+ */
+class <?= $class_name ?> extends Controller
+{
+    /**
+     * @Route("/", name="index")
+     *
+     * @return Response
+     */
+    public function index()
+    {
+        $<?= $entity_var_plural; ?> = $this->getDoctrine()
+            ->getRepository(<?= $entity_class_name; ?>::class)
+            ->findAll();
+
+        return $this->render('<?= $route_name; ?>/index.html.twig', ['<?= $entity_var_plural; ?>' => $<?= $entity_var_plural; ?>]);
+    }
+
+    /**
+     * @Route("/new", name="new")
+     * @Method({"GET", "POST"})
+     */
+    public function new(Request $request)
+    {
+        $<?= $entity_var_singular; ?> = new <?= $entity_class_name; ?>();
+        $form = $this->createForm(<?= $form_class_name; ?>::class, $<?= $entity_var_singular; ?>);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($<?= $entity_var_singular; ?>);
+            $em->flush();
+
+            return $this->redirectToRoute('<?= $route_name; ?>_edit', ['<?= $entity_identifier; ?>' => $<?= $entity_var_singular; ?>->get<?= ucfirst($entity_identifier); ?>()]);
```

Comment:
What do you think about flash messages? It would be great to add them... but we don't actually know whether or not the user is printing flash messages in `base.html.twig`. So, it's tricky... I'm not sure personally...

Reply:
I have an idea about it. I'll push it soon